src/modules/access-wallet/hardware/components/AccessWalletNetworkAddresses.vue

```diff
@@ -22,9 +22,6 @@
           <div class="headline font-weight-bold">
```

Comment:
can you add "mb-5" class to this div (otherwise there is no space once you removed bottom p